.github/wiki/Test-The-Apply-Action-On-Development.md.md

```diff
@@ -0,0 +1,47 @@
+# Test the apply action on development
```

Comment:
![Pasted_Image_10_03_2022__10_51](https://user-images.githubusercontent.com/11751745/157585182-4f228490-2891-4020-a4a0-2301af4b5f80.png)

Reply:
Fixed in f6e095a